docs/optimizing-flatlist-configuration.md

```diff
@@ -25,9 +25,9 @@ Here are a list of props that can help to improve `FlatList` performance:
 
 ### removeClippedSubviews
 
-| Type    | Default |
-| ------- | ------- |
-| Boolean | False   |
+| Type    | Default                     |
+| ------- | --------------------------- |
+| Boolean | False (iOS), True (Android) |
```

Comment:
```suggestion
| Type    | Default                              |
| ------- | ------------------------------------ |
| Boolean | `true` on Android, otherwise `false` |
```

Let's make this description more inline with the current code:
* https://github.com/facebook/react-native/blob/main/Libraries/Lists/FlatList.js#L158